src/sentry/event_manager.py

```diff
@@ -871,6 +872,12 @@ def _save_aggregate(self, event, hashes, release, **kwargs):
                     **kwargs
                 ), True
 
+            metrics.incr(
```

Comment:
i think we have an object.created metric fwiw

Reply:
We do, but it is not per-platform.